documentation/modules/adding-source-provider.adoc

```diff
@@ -78,7 +78,7 @@ ifdef::vmware[]
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider.
-* *URL*: vCenter host name or IP address - if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
+* *URL*: API endpoint of the vCenter on which the source VM is mounted, for example, `https://vCenter-host-example.com/sdk`. If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
```

Comment:
as I commented on #745, better to use `vCenter SDK endpoint`

Reply:
Fixed.